typescript/lib/networks/network_context.ts

```diff
@@ -0,0 +1,604 @@
+/*
+ * Copyright (c) 2020 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+import { ApiContainerServiceClient } from "../..//kurtosis_core_rpc_api_bindings/api_container_service_grpc_pb";
+import { LoadLambdaArgs, GetLambdaInfoArgs, RegisterStaticFilesArgs, RegisterStaticFilesResponse, RegisterFilesArtifactsArgs, PortBinding, RegisterServiceArgs, RegisterServiceResponse, StartServiceArgs, GetServiceInfoArgs, GetServiceInfoResponse, RemoveServiceArgs, PartitionConnectionInfo, PartitionServices, PartitionConnections, RepartitionArgs, WaitForEndpointAvailabilityArgs, ExecuteBulkCommandsArgs, StartServiceResponse, GetLambdaInfoResponse } from "../..//kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { LambdaID, LambdaContext } from "../modules/lambda_context";
+import { ServiceID } from "../services/service";
+import { ServiceContext, GeneratedFileFilepaths } from "../services/service_context";
+import { StaticFileID, FilesArtifactID, ContainerCreationConfig } from "../services/container_creation_config"; 
+import { ContainerRunConfig } from "../services/container_run_config";
+import { newLoadLambdaArgs, newLambdaInfoArgs, newRegisterStaticFilesArgs, newRegisterFilesArtifactsArgs, newRegisterServiceArgs, newStartServiceArgs, newGetServiceInfoArgs, newRemoveServiceArgs, newPartitionServices, newPartitionConnections, newRepartitionArgs, newWaitForEndpointAvailabilityArgs, newExecuteBulkCommandsArgs } from "../constructor_calls";
+import { okAsync, errAsync, ResultAsync, ok, err, Result } from "neverthrow";
+import * as log from "loglevel";
+import * as path from "path";
+import * as fs from 'fs';
+import * as fsPromises from "fs/promises";
+import * as grpc from "grpc";
+import * as google_protobuf_empty_pb from "google-protobuf/google/protobuf/empty_pb";
+
+export type PartitionID = string;
+
+// This will always resolve to the default partition ID (regardless of whether such a partition exists in the network,
+//  or it was repartitioned away)
+const defaultPartitionId: PartitionID = "";
```

Comment:
Oo forgot to mention - Java/Typescript's convention is UPPER_SNAKE_CASE'd constants (unlike Go's, which is lowerCamelCase, which I disagree with because IMO it makes it look too much like a normal variable)